CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Release History
 
+## [v1.2.0](unreleased)
+
+Enhancement:
+
+- Updated code to use Test-VCFConnection instead of Test-Connection 
+- Updated code to use Test-EsxiConnection instead of Test-NetConnection 
```

Comment:
```suggestion

- Updated to use `Test-EsxiConnection` instead of `Test-NetConnection` where applicable. [GH-45](https://github.com/vmware/powershell-module-for-vmware-cloud-foundation-certificate-management/pull/45)
```

Reply:
Resolved in https://github.com/vmware/powershell-module-for-vmware-cloud-foundation-certificate-management/pull/45/commits/48ca113f5f0d3edc9163c227b3b3871f636f2e81.